.github/workflows/linux.yml

```diff
@@ -21,7 +21,7 @@ jobs:
          run: |
            set -e -x 
            sudo apt-get update
-           sudo apt-get install -y libtinfo-dev zlib1g-dev build-essential libedit-dev libxml2-dev
+           sudo apt-get install -y libtinfo-dev zlib1g-dev build-essential libedit-dev libxml2-dev nasm
```

Comment:
One last question: is nasm needed to TVM only?

Reply:
Yes, this dependency appears when adding intel/ipp-crypto

Reply:
I'm wondering if there is a way to only install it when needed.